test/correctness/CMakeLists.txt

```diff
@@ -1,324 +1,378 @@
-        plain_c_includes.c
-        popc_clz_ctz_bounds.cpp
-        predicated_store_load.cpp
-        prefetch.cpp
-        print.cpp
-        process_some_tiles.cpp
-        pseudostack_shares_slots.cpp
-        python_extension_gen.cpp
-        random.cpp
-        realize_larger_than_two_gigs.cpp
-        realize_over_shifted_domain.cpp
-        reduction_chain.cpp
-        reduction_non_rectangular.cpp
-        reduction_schedule.cpp
-        register_shuffle.cpp
-        reorder_rvars.cpp
-        reorder_storage.cpp
-        require.cpp
-        reschedule.cpp
-        reuse_stack_alloc.cpp
-        rfactor.cpp
-        round.cpp
-        saturating_casts.cpp
-        scatter.cpp
-        set_custom_trace.cpp
-        shared_self_references.cpp
-        shifted_image.cpp
-        side_effects.cpp
-        simd_op_check.cpp
-        simd_op_check_hvx.cpp
-        simplified_away_embedded_image.cpp
-        simplify.cpp
-        skip_stages.cpp
-        skip_stages_external_array_functions.cpp
-        skip_stages_memoize.cpp
-        sliding_backwards.cpp
-        sliding_reduction.cpp
-        sliding_window.cpp
-        sort_exprs.cpp
-        specialize.cpp
-        specialize_to_gpu.cpp
-        split_by_non_factor.cpp
-        split_fuse_rvar.cpp
-        split_reuse_inner_name_bug.cpp
-        split_store_compute.cpp
-        stack_allocations.cpp
-        stencil_chain_in_update_definitions.cpp
-        stmt_to_html.cpp
-        storage_folding.cpp
-        store_in.cpp
-        stream_compaction.cpp
-        strict_float_bounds.cpp
-        strict_float.cpp
-        strided_load.cpp
-        target.cpp
-        thread_safety.cpp
-        tracing_bounds.cpp
-        tracing_broadcast.cpp
-        tracing.cpp
-        tracing_stack.cpp
-        transitive_bounds.cpp
-        trim_no_ops.cpp
-        truncated_pyramid.cpp
-        tuple_partial_update.cpp
-        tuple_reduction.cpp
-        tuple_select.cpp
-        tuple_undef.cpp
-        tuple_update_ops.cpp
-        two_vector_args.cpp
-        undef.cpp
-        uninitialized_read.cpp
-        unique_func_image.cpp
-        unroll_dynamic_loop.cpp
-        unrolled_reduction.cpp
-        unsafe_dedup_lets.cpp
-        unsafe_promises.cpp
-        unused_func.cpp
-        update_chunk.cpp
-        vector_bounds_inference.cpp
-        vector_cast.cpp
-        vector_extern.cpp
-        vectorized_gpu_allocation.cpp
-        vectorized_initialization.cpp
-        vectorized_load_from_vectorized_allocation.cpp
-        vectorized_reduction_bug.cpp
-        vectorize_guard_with_if.cpp
-        vectorize_mixed_widths.cpp
-        vectorize_varying_allocation_size.cpp
-        vector_math.cpp
-        vector_print_bug.cpp
-        widening_reduction.cpp
-        )
+      SOURCES
+      align_bounds.cpp
+      argmax.cpp
+      assertion_failure_in_parallel_for.cpp
+      async_copy_chain.cpp
+      async.cpp
+      async_device_copy.cpp
+#      atomics.cpp
```

Comment:
Why is this commented out?

Reply:
Because it depends on knowing the LLVM_VERSION Halide is being built against and I haven't gotten that properly propagated through yet. Adding that to the top-level TODOs on this PR.